misc/ecology/occupancy/occupancy.stan

```diff
@@ -34,9 +34,9 @@ model {
   // likelihood
   for (r in 1:R) {
     if (sum(y[r]) > 0)
-      increment_log_prob(log_psi1 + bernoulli_log(y[r],p));
+      target += log_psi1 + bernoulli_lpmf(y[r] | p);
     else
-      increment_log_prob(log_sum_exp(log_psi1 + bernoulli_log(y[r],p),
-                                     log1m_psi1));
+      target += log_sum_exp(log_psi1 + bernoulli_lpmf(y[r] | p),
+			    log1m_psi1);;
```

Comment:
Thanks.  This has a stray `;` at the end and ideally the `log1m` should line up under the `log`.